samples/cpp/benchmark_app/main.cpp

```diff
@@ -738,9 +793,16 @@ int main(int argc, char* argv[]) {
             // --------------------------------------------------------
             next_step();
             startTime = Time::now();
+            auto compileModelMemStart = getPeakMemoryUsage();
             compiledModel = core.compile_model(model, device_name, device_config);
+            auto compileModelMemEnd = getPeakMemoryUsage();
             duration_ms = get_duration_ms_till_now(startTime);
```

Comment:
I think `getPeakMemoryUsage()` shouldn't be captured by `get_duration_ms_till_now()`. It's better to capture `get_duration_ms_till_now()` in `getPeakMemoryUsage()` instead because `get_duration_ms_till_now()` is simpler and thus more predictable.

Reply:
Moved, please check

Reply:
This is applicable to python too

Reply:
Moved

Reply:
You shouldn't capture `logger.inf()`